ci/playbooks/kuttl/run-kuttl-tests.yml

```
@@ -61,3 +61,30 @@
     {{ item }} > {{ cifmw_artifacts_basedir }}/logs/cmd_after_{{ operator }}_kuttl.log
   loop: "{{ commands_after_kuttl_run }}"
   ignore_errors: true
+
+# Same of the xml files may not be the JUnitXML results, but getting the exact file
+# name (parsing the KUTTL configuration) may be too cumbersome.
+- name: 'Find the generated JUnitXML files'
```

nit: This project normally doesn't quote task names

```suggestion
- name: Find the generated JUnitXML files
```

---

And still the rest of the tasks in this file use quotes. So should I chose consistency with the project or file-level consistency?

---

ok, ok